EIPS/eip-4361-1.md

```diff
@@ -0,0 +1,238 @@
+---
+eip: TBD
+title: SIWE CapGrok Extension
+description: 
+author: Oliver Terbu (@awoie), Jacob Ward, Charles Lehner, Sam Gbafa, Wayne Chang (@wyc), Gregory Rocco (@obstropolos)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-20
+requires: 4361
+---
+
+## Abstract
+Sign-In with Ethereum (SIWE) describes how Ethereum accounts authenticate with off-chain services (relying parties) by signing a standard message format parameterized by scope, session details, and security mechanisms (e.g., a nonce). This specification describes a mechanism on top of SIWE to give informed consent to delegate capabilities with a certain extensible scope mechanism to an authorized delegee. The way how a delegee authenticates against the target resource is out of scope of this specification and depends on the implementation of the target resource. This specification is fully-backwards compatible with EIP-4361.
+
+## Motivation
+
+(non-normative)
+
+While SIWE focuses on authenticating the Ethereum account against the service (relying party or SIWE client) initiating the SIWE flow, there is no canonical way to interact with a third-party service (resource service) on behalf of the authenticated Ethereum account. For example, a service might want to interact with a (decentralized) protocol that is performining data storage or key management on behalf of the Ethereum account. This specification introduces a mechanism, that allows the service (or more generally a delegee) to combine authentication and authorization of such while preserving security and optimizing UX.
+
+Note, this approach is a similar mechanism to combining OpenID Connect (SIWE auth) and OAuth2 (SIWE CapGrok). Resource service implementer's should not consider CapGroks as bearer tokens but instead require to authenticate the delegee in addition. The process of authenticating the delegee against the resource service is out of scope of this specification and can be done in various different ways.
+
+SIWE CapGroks unlock integration of (decentralized) protocols and/or APIs for developers by reducing user friction, onchain state and increasing security by introducing informed consent and deterministic capability objects.
+
+This specification has three different audiences:
+- Web3 application developers that want to integrate CapGroks to interact with Web3 protocols and APIs.
+- Web3 protocol or API developers that want to learn how to define their own CapGroks.
+- Wallet implementers that want to improve the UI for capgroks.
```

Comment:
In my opinion "decentralised" or "Web3" protocols/APIs is an unnecessary narrowing of the scope.
```suggestion
- Web3 application developers that want to integrate CapGroks to authenticate with any protocols and APIs that support object capabilities.
- Protocol or API developers that want to learn how to define their own CapGroks.
- Wallet implementers that want to improve the UI for CapGroks.
```